index.js

```diff
@@ -61,6 +89,11 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 5", "color: red")
 
+
+// function updateGrade(student, grade){
+//   student1.name = "Duane";
+//   student1.grade = grade;
+// }
```

Comment:
For this function, there's no need to assign the name of the student - the only thing we're modifying is the grade: 

```js
function updateGrade(student, grade){
  student1.grade = grade;
}
```